config/routes.ts

```diff
@@ -16,10 +16,30 @@ export default [
           },
         ],
       },
+      {
+        path: "/models",
```

Comment:
let's have here the id of the first datasource

/models/:datasource_id

because currently it shows 404 and then redirect – it's confusing